disconnect-entitylist.json

```diff
@@ -1,12 +1,20 @@
 {
     "365Media": {
         "properties": [
-            "365dm.com",
-            "365media.com"
+            "aggregateintelligence.com"
         ],
         "resources": [
             "365dm.com",
-            "365media.com"
+            "365media.com",
+            "aggregateintelligence.com"
+        ]
+    },
+	"4mads": {
```

Comment:
We should eliminate these entries (i.e. where there is a single resource and a single property). They don't do anything because a domain is never cross-origin with itself.

It would help reduce the bloat on this list.

Reply:
By "eliminate", I mean in the script that generates the actual binary list. It's fine if these entries are in this json list.

Reply:
Can you file on https://github.com/mozilla-services/shavar-list-creation ?

Reply:
Filed at https://github.com/mozilla-services/shavar-list-creation/issues/49

Reply:
Should we block this PR on that?

Reply:
I think so because if we're not filtering those out, it will likely lead to a significant reduction in the size of the entity list.